keywords.md

```diff
@@ -32,6 +35,8 @@ However, same suffixes result in same values for different properties. The follo
 }
 ```
 
+### Instant, date time and time stamp calculation
```

Comment:
timestamp instead of time stamp?

Reply:
Hehe, well, spell checker complains about timestamp but accepts time stamp, though I think it depends on the spell checker.. :+1:

Reply:
Too many places to change for now and that PR I think, but will keep in mind... :)